tunnistamo/settings.py

```diff
@@ -77,6 +77,9 @@
     SOCIAL_AUTH_YLETUNNUS_APP_KEY=(str, ""),
     SOCIAL_AUTH_YLETUNNUS_SECRET=(str, ""),
 
+    # Value must be either "test" or "production"
+    SOCIAL_AUTH_YLETUNNUS_ENVIRONMENT=(str, ""),
+
```

Comment:
This looks to be the correct abstraction level.

Reply:
Although, maybe default to production?

Reply:
On the other hand, there will be more Tunnistamos connected to test than to prod. Hmm, which one would be more secure by default? Or insecure? Having production connected to test by accident or vice versa? This forces one to make the choice.

Reply:
@mikkokeskinen Which default? I'm thinking production, since our no1 aim is to protect the production instance and will never want to enable test there by accident.

Reply:
I would set it to "production" too.

Reply:
Fixed in 48566927e92106c400af922b8361f0cf4a61d65b